site/content/community/02-roadmap.md

```diff
@@ -13,9 +13,9 @@ One or more release candidate(s) (RC) builds will be made available prior to eac
 
 ## Project Roadmap Themes
 
-### Cluster quickstart with GitOps
+### Cluster Quick Start with gitops
```

Comment:
If "Quick Start" needs to be capitalised in this title, we should also consider capitalising gitops:

```suggestion
### Cluster Quick Start with Gitops
```